fmriprep/utils/sentry.py

```diff
@@ -180,3 +180,27 @@ def _chunks(string, length=CHUNK_SIZE):
     """
     return (string[i:i + length]
             for i in range(0, len(string), length))
+
+
+def ping_migas(status='pending'):
```

Comment:
Also, let's go ahead and type this...

```suggestion
def ping_migas(status: str='pending') -> None:
```

I'm not sure what `res` is on L200, but we never use the return value. I'm guessing it's a `dict`, so if we think we want this function to return a `dict`, then let's change L192 to `return {}` and set:

```suggestion
def ping_migas(status: str='pending') -> dict:
```

Reply:
I think @mgxd  said that type annotations are not very well supported by Python 3.7, right?

Reply:
nope, they are supported - they are just much more verbose or require `typing-extensions`